.eslintrc

```diff
@@ -0,0 +1,26 @@
+{
+  "extends": "airbnb-base",
+  "env": {
+    "es6": true,
+    "browser": true,
+    "jasmine": true,
+    "node": true
+  },
+  "rules": {
+    "import/prefer-default-export": 0,
+    "import/no-extraneous-dependencies": 0,
+    "import/no-dynamic-require": 1,
```

Comment:
yes, by default it is configured to error lever in airbnb preset this line says - to show just a warning to note that this should be eventually changed(currently we have such require in writeReport.js, and I think we will change that when adding new cli interface)